doc/tutorials/contribute/Contributor-Guide.md

```diff
@@ -365,6 +364,34 @@ cd api
 npm run test:here <folderName> <describeString>
 ```
 
+### Git-Secrets
```

Comment:
suggestion: put this section on the top of the page

Reply:
done